### DISCUSSION TITLE
How do I edit the "need help" tooltip?

### DISCUSSION THREAD

**Discussion by moose999** [LiveHelperChat/livehelperchat#2127] [General]
I'd like to change the text / image on this popup:
![image](https://github.com/user-attachments/assets/f616c728-0bb8-409f-abed-4b2249c294cf)

What's the best way to do that?

Many thanks!

**Comment by remdex** [comment]
Just create a widget theme in back office and you can edit text directly in back office.

**Comment by moose999** [comment]
Many thanks!!
